### DISCUSSION TITLE
Introduce arrange displays quick options from menubar

### DISCUSSION THREAD

**Discussion by AkarshSatija** [MonitorControl/MonitorControl#1591] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Currently arranging displays requires getting to settings and thus into arrange tab

### Describe the solution you'd like

Providing a way to rearrange displays' arrangement quickly from the menubar

### Describe alternatives you've considered

N/A

**Comment by waydabber** [comment]
I added this feature to [BetterDisplay](https://github.com/waydabber/BetterDisplay) ("Move Display" menu). It lets you quickly snap a display next to an other one (adjusted to a corner or centered). The app also has a more advanced layout protection feature under Settings/Groups that keeps relative positions.
